docs-src/cloud-context/saml-finish.md

```diff
@@ -0,0 +1,19 @@
+---
+id: saml-finish
+title: How to finish your SAML configuration
+sidebar_label: Finish SAML configuration
+description: To finish your SAML configuration, send the sign-in URL and certificate to us and then test your connection.
```

Comment:
They need to include the email domain they want to map to their saml connection. i.e. for us it is "temporal.io".

They have the option to provide us multiple domains.

Reply:
nit: might want to name certificate the "X509 Signing Certificate"